packages/tasks_panel/src/services/tasks-provider.ts

```diff
@@ -35,16 +35,14 @@ export class TasksProvider implements ITasksProvider, ITasksEventHandler {
           continue;
         }
 
-        const configuredContributedTasks = filter(configuredTasks, (_) => supportedTypes.includes(_.type));
+        const configuredContributedTasks = cloneDeep(configuredTasks);
```

Comment:
Maybe add a comment on what's the issue with the index and why it has to start with "1" and not "0"?